resources/views/participant/welcome.blade.php

```diff
@@ -1,11 +1,57 @@
 @extends('layouts.participant_master')
 
 @section('content')
-    <div class="title m-b-md">
-        Spot The Bug Challenge
-    </div>
     <p>Logged in as {{Auth::guard('participant')->user()->email}}</p>
 
+    <div id="TopDiv">
+        <div id="ChallengeContainer" class="container">
```

Comment:
If 1 challenge is visible the box shows an "ugly" end of the box. add "padding 50px" to create a nicer view
![image](https://user-images.githubusercontent.com/9818590/33174271-a961df40-d057-11e7-904a-c66b91cd631a.png)